nanostack-event-loop/platform/arm_hal_timer.h

```diff
@@ -48,6 +48,31 @@ extern void platform_timer_disable(void);
  * \return 50us time slot remaining
  */
 extern uint16_t platform_timer_get_remaining_slots(void);
+/**
```

Comment:
Maybe bracket this section in the ifdef? Needs to be clear this only needs to be provided by the platform if that define is active.